content/docs-v1/v1-tutorial.md

```diff
@@ -45,6 +45,7 @@ In addition to telling Quire what to do, the command-line shell can also be used
 3. Type `cd` followed by the name of a directory you want to navigate to. For example, `cd Downloads` will take you into the Downloads directory.
 4. Run `ls` again to view the contents of that folder.
 5. Pull up your Finder or File Explorer to compare the listed files.
+6. Type `cd` again to go back to your home directory.
```

Comment:
on WIndows, it **has** to be `cd ~` to go to the home directory. `cd ~` also works on Mac so maybe that should be used here.